app/src/main/res/layout/share_episode_dialog.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:orientation="vertical"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:gravity="center"
+    android:padding="16dp">
+
+    <RadioGroup
+        android:id="@+id/share_dialog_radio_group"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_marginBottom="16dp"
+        android:orientation="vertical">
+
+        <RadioButton android:id="@+id/share_episode_website_radio"
+            android:layout_width="wrap_content"
```

Comment:
If you set the width to `match_parent`, the buttons get easier to tap (while looking exactly the same).

Reply:
Added :)